Error on iOS with v0.6.0

When building for iOS I'm getting this:

[ERROR] The application has crashed with an uncaught exception 'NSInvalidArgumentException'.
[ERROR] Reason:
[ERROR] -[NSNull paintContext:bounds:]: unrecognized selector sent to instance 0x107862180
[ERROR] Stack trace:
[ERROR] 0   CoreFoundation                      0x00000001075a5af3 __exceptionPreprocess + 147
[ERROR] 1   libobjc.A.dylib                     0x0000000106b3a141 objc_exception_throw + 48
[ERROR] 2   CoreFoundation                      0x0000000107615134 -[NSObject(NSObject) doesNotRecognizeSelector:] + 132
[ERROR] 3   CoreFoundation                      0x000000010752c840 ___forwarding___ + 1024
[ERROR] 4   CoreFoundation                      0x000000010752c3b8 _CF_forwarding_prep_0 + 120
[ERROR] 5   App                      0x00000001009e8547 -[TiGradientLayer drawInContext:] + 183
[ERROR] 6   QuartzCore                          0x0000000105685ae5 CABackingStoreUpdate_ + 2505
[ERROR] 7   QuartzCore                          0x000000010576f0fa ___ZN2CA5Layer8display_Ev_block_invoke + 61
[ERROR] 8   QuartzCore                          0x000000010576edf3 _ZN2CA5Layer8display_Ev + 1633
[ERROR] 9   QuartzCore                          0x000000010576331d _ZN2CA5Layer17display_if_neededEPNS_11TransactionE + 315
[ERROR] 10  QuartzCore                          0x00000001057633ab _ZN2CA5Layer28layout_and_display_if_neededEPNS_11TransactionE + 35
[ERROR] 11  QuartzCore                          0x00000001056f2e92 _ZN2CA7Context18commit_transactionEPNS_11TransactionE + 294
[ERROR] 12  QuartzCore                          0x000000010571f130 _ZN2CA11Transaction6commitEv + 468
[ERROR] 13  QuartzCore                          0x000000010571fb37 _ZN2CA11Transaction17observer_callbackEP19__CFRunLoopObservermPv + 115
[ERROR] 14  CoreFoundation                      0x000000010754b717 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 23
[ERROR] 15  CoreFoundation                      0x000000010754b687 __CFRunLoopDoObservers + 391
[ERROR] 16  CoreFoundation                      0x0000000107530720 __CFRunLoopRun + 1200
[ERROR] 17  CoreFoundation                      0x0000000107530016 CFRunLoopRunSpecific + 406
[ERROR] 18  GraphicsServices                    0x0000000108eb5a24 GSEventRunModal + 62
[ERROR] 19  UIKit                               0x00000001025e90d4 UIApplicationMain + 159
.....

Tested on version 0.6.0 and it's perfect. I think could be related to the gradient changes introduced in latest version and maybe related to https://jira.appcelerator.org/browse/TIMOB-14571?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel

Thanks,
Yanko

--- a/README.md
+++ b/README.md
@@ -68,6 +68,7 @@ $.progress.animate({
 ### Properties
 * `progressColor` _(String)_ - the color of the progress layer (default: `"#000"`)
 * `progressBackgroundColor` _(String)_ - the color of the outstanding part of progress (default: `"#666"` 😈)
+* `progressBackgroundGradient` _(Gradient)_ - the progressBackground as [gradient](http://docs.appcelerator.com/platform/latest/#!/api/Gradient)
 * `progressWidth` _(Number)_ - the width of the progress (if not defined, progress becomes a cake)
 * `formatValue`_(Function)_ - formats the value displayed in the center
 * `showText` _(Boolean)_ - whether the current progress value should be displayed (default: `false`)

--- a/de.manumaticx.circularprogress/controllers/widget.js
+++ b/de.manumaticx.circularprogress/controllers/widget.js
@@ -9,7 +9,7 @@ var defaults = {
   backgroundColor: '#fff',
   progressColor: '#000',
   progressBackgroundColor: '#666',
-  progressBackgroundGradient: false,
+  progressBackgroundGradient: undefined,
   formatValue: function(v){return v;},
   showText: false
 };
@@ -70,9 +70,9 @@ function createView(_args){
   $.progresslayer.backgroundColor = options.progressColor;
   $.progresslayer.backgroundGradient = options.progressGradient;
   $.leftlayer.backgroundColor = options.progressBackgroundColor || options.backgroundColor;
-  $.leftlayer.backgroundGradient = options.progressBackgroundGradient || options.backgroundColor;
+  $.leftlayer.backgroundGradient = options.progressBackgroundGradient;
   $.innerrotationlayer.backgroundColor = options.progressBackgroundColor || options.backgroundColor;
-  $.innerrotationlayer.backgroundGradient = options.progressBackgroundGradient || options.backgroundColor;
+  $.innerrotationlayer.backgroundGradient = options.progressBackgroundGradient;
   $.rightlayer.backgroundColor = options.progressColor;
   $.rightlayer.backgroundGradient = options.progressGradient;
   $.centerlayer.backgroundColor = options.backgroundColor;

--- a/de.manumaticx.circularprogress/widget.json
+++ b/de.manumaticx.circularprogress/widget.json
@@ -3,7 +3,7 @@
 	"name": "Circular Progress",
 	"description" : "A Circular Progressbar",
 	"author": "Manuel Lehner (manumaticx@gmail.com)",
-	"version": "0.6.0",
+	"version": "0.6.1",
 	"copyright":"Copyright (c) 2014",
 	"license":"MIT",
 	"min-alloy-version": "1.3",

